android/app/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,11 @@
 
     <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+    <uses-permission android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />
```

Comment:
Why this is needed?

Reply:
Using we can listen for location updates and suggest a location update (Future feature)

Reply:
I think then we can delete it since we don't use it, background location isn't favorable for users